### PR TITLE
add timezone and update logging config setup

### PR DESCRIPTION
**Pull request by jihyunbak**
small changes

### CODE REVIEW

File: nsds_lab_to_nwb/nwb_builder.py [hunk 1]
```diff
@@ -19,15 +20,19 @@
 from nsds_lab_to_nwb.components.stimulus.stimulus_originator import StimulusOriginator
 from nsds_lab_to_nwb.components.tdt.tdt_originator import TdtOriginator
 
+# PWD = os.path.dirname(os.path.abspath(__file__))
+# logging.config.fileConfig(fname=str(PWD) + '/logging.conf', disable_existing_loggers=False)
```
Comment (JesseLivezey) [inline nsds_lab_to_nwb/nwb_builder.py:24]: Should these still be here? Remove?

File: scripts/generate_nwb.py [hunk 2]
```diff
@@ -6,6 +7,9 @@
 PWD = os.path.dirname(os.path.abspath(__file__))
 USER_HOME = os.path.expanduser("~")
```
Comment (JesseLivezey) [inline scripts/generate_nwb.py:8]: Is this used?

File: scripts/generate_nwb.py [hunk 3]
```diff
@@ -6,6 +7,9 @@
...
 
+logging.config.fileConfig(fname=str(PWD) + '/../nsds_lab_to_nwb/logging.conf', disable_existing_loggers=False)
```
Comment (JesseLivezey) [inline scripts/generate_nwb.py:10]: I think we'll eventually want to do this in a different way, I'll make an issue.

**Review by JesseLivezey — CHANGES_REQUESTED** [review]

File: nsds_lab_to_nwb/nwb_builder.py (hunk 1 above)
Comment (jihyunbak) [reply nsds_lab_to_nwb/nwb_builder.py]: Removed, thanks!

File: scripts/generate_nwb.py (hunk 2 above)
Comment (jihyunbak) [reply scripts/generate_nwb.py:8]: If you are referring to PWD and USER_HOME, these are used to set the output and metadata paths below. Could be improved by setting up better central paths in later updates.

File: scripts/generate_nwb.py (hunk 3 above)
Comment (jihyunbak) [reply scripts/generate_nwb.py:10]: Sounds good, look forward to seeing your solution - if there's a better way to apply fileConfig that would be nice to know.

**Review by JesseLivezey — APPROVED** [review]
LGTM

**Pull request merged**